Misc/NEWS.d/next/Library/2025-03-18-14-18-06.gh-issue-91349.Qrnmxt.rst

```diff
@@ -0,0 +1,3 @@
+Expose the crc32 function from the lzma library. It is faster than the crc32
+function from the zlib library. Update zipfile to detect and use lzma.crc32
+zlic.crc32 binascii.crc32, in order of preference.
```

Comment:
Are there rough benchmark numbers? We should try and avoid an unqualified "it's faster" claim.

There are also no changes to `zipfile` in this PR?

```suggestion
Expose the crc32 function from the lzma library as :func:`lzma.crc32`.
It is faster than the crc32 function from the zlib library.
Update :mod:`zipfile` to detect and use :func:`lzma.crc32`, :func:`zlic.crc32`,
and :func:`binascii.crc32`, in descending order of preference.
```